org.apache.accumulo.master.replication.WorkMaker.run : $r11 = r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.core.client.Connector conn> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $z0 = staticinvoke <org.apache.accumulo.core.replication.ReplicationTable: boolean isOnline(org.apache.accumulo.core.client.Connector)>($r11)
org.apache.accumulo.master.replication.WorkMaker.run : $z0 = staticinvoke <org.apache.accumulo.core.replication.ReplicationTable: boolean isOnline(org.apache.accumulo.core.client.Connector)>($r11) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : if $z0 != 0
org.apache.accumulo.master.replication.WorkMaker.run : if $z0 != 0 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r33 = <org.apache.accumulo.master.replication.WorkMaker: org.slf4j.Logger log>
org.apache.accumulo.master.replication.WorkMaker.run : if $z0 != 0 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : r1 = staticinvoke <org.apache.accumulo.core.trace.Trace: org.apache.accumulo.core.trace.Span start(java.lang.String)>("replicationWorkMaker")
org.apache.accumulo.master.replication.WorkMaker.run : r1 = staticinvoke <org.apache.accumulo.core.trace.Trace: org.apache.accumulo.core.trace.Span start(java.lang.String)>("replicationWorkMaker") >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r12 = r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.core.client.Connector conn>
org.apache.accumulo.master.replication.WorkMaker.run : r1 = staticinvoke <org.apache.accumulo.core.trace.Trace: org.apache.accumulo.core.trace.Span start(java.lang.String)>("replicationWorkMaker") >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r1.<org.apache.accumulo.core.trace.Span: void stop()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : throw $r39
org.apache.accumulo.master.replication.WorkMaker.run : $r12 = r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.core.client.Connector conn> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r36 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r12 = r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.core.client.Connector conn> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r12 = r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.core.client.Connector conn> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : r2 = staticinvoke <org.apache.accumulo.core.replication.ReplicationTable: org.apache.accumulo.core.client.Scanner getScanner(org.apache.accumulo.core.client.Connector)>($r12)
org.apache.accumulo.master.replication.WorkMaker.run : r2 = staticinvoke <org.apache.accumulo.core.replication.ReplicationTable: org.apache.accumulo.core.client.Scanner getScanner(org.apache.accumulo.core.client.Connector)>($r12) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r36 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : r2 = staticinvoke <org.apache.accumulo.core.replication.ReplicationTable: org.apache.accumulo.core.client.Scanner getScanner(org.apache.accumulo.core.client.Connector)>($r12) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : r2 = staticinvoke <org.apache.accumulo.core.replication.ReplicationTable: org.apache.accumulo.core.client.Scanner getScanner(org.apache.accumulo.core.client.Connector)>($r12) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r13 = r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.core.client.BatchWriter writer>
org.apache.accumulo.master.replication.WorkMaker.run : $r13 = r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.core.client.BatchWriter writer> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : if null != $r13
org.apache.accumulo.master.replication.WorkMaker.run : $r13 = r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.core.client.BatchWriter writer> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : if null != $r13 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r31 = r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.core.client.Connector conn>
org.apache.accumulo.master.replication.WorkMaker.run : if null != $r13 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : goto [?= staticinvoke <org.apache.accumulo.core.replication.ReplicationSchema$StatusSection: void limit(org.apache.accumulo.core.client.ScannerBase)>(r2)]
org.apache.accumulo.master.replication.WorkMaker.run : if null != $r13 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : goto [?= staticinvoke <org.apache.accumulo.core.replication.ReplicationSchema$StatusSection: void limit(org.apache.accumulo.core.client.ScannerBase)>(r2)] >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : staticinvoke <org.apache.accumulo.core.replication.ReplicationSchema$StatusSection: void limit(org.apache.accumulo.core.client.ScannerBase)>(r2)
org.apache.accumulo.master.replication.WorkMaker.run : goto [?= staticinvoke <org.apache.accumulo.core.replication.ReplicationSchema$StatusSection: void limit(org.apache.accumulo.core.client.ScannerBase)>(r2)] >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : staticinvoke <org.apache.accumulo.core.replication.ReplicationSchema$StatusSection: void limit(org.apache.accumulo.core.client.ScannerBase)>(r2) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r14 = new org.apache.hadoop.io.Text
org.apache.accumulo.master.replication.WorkMaker.run : staticinvoke <org.apache.accumulo.core.replication.ReplicationSchema$StatusSection: void limit(org.apache.accumulo.core.client.ScannerBase)>(r2) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r14 = new org.apache.hadoop.io.Text >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : specialinvoke $r14.<org.apache.hadoop.io.Text: void <init>()>()
org.apache.accumulo.master.replication.WorkMaker.run : $r14 = new org.apache.hadoop.io.Text >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : specialinvoke $r14.<org.apache.hadoop.io.Text: void <init>()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : r4 = interfaceinvoke r2.<org.apache.accumulo.core.client.Scanner: java.util.Iterator iterator()>()
org.apache.accumulo.master.replication.WorkMaker.run : specialinvoke $r14.<org.apache.hadoop.io.Text: void <init>()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : r4 = interfaceinvoke r2.<org.apache.accumulo.core.client.Scanner: java.util.Iterator iterator()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()
org.apache.accumulo.master.replication.WorkMaker.run : r4 = interfaceinvoke r2.<org.apache.accumulo.core.client.Scanner: java.util.Iterator iterator()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : if $z3 == 0
org.apache.accumulo.master.replication.WorkMaker.run : $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : if $z3 == 0 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r15 = interfaceinvoke r4.<java.util.Iterator: java.lang.Object next()>()
org.apache.accumulo.master.replication.WorkMaker.run : if $z3 == 0 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r1.<org.apache.accumulo.core.trace.Span: void stop()>()
org.apache.accumulo.master.replication.WorkMaker.run : if $z3 == 0 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r1.<org.apache.accumulo.core.trace.Span: void stop()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : goto [?= return]
org.apache.accumulo.master.replication.WorkMaker.run : goto [?= return] >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : return
org.apache.accumulo.master.replication.WorkMaker.run : $r15 = interfaceinvoke r4.<java.util.Iterator: java.lang.Object next()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : r5 = (java.util.Map$Entry) $r15
org.apache.accumulo.master.replication.WorkMaker.run : $r15 = interfaceinvoke r4.<java.util.Iterator: java.lang.Object next()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : r5 = (java.util.Map$Entry) $r15 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r16 = interfaceinvoke r5.<java.util.Map$Entry: java.lang.Object getKey()>()
org.apache.accumulo.master.replication.WorkMaker.run : r5 = (java.util.Map$Entry) $r15 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r16 = interfaceinvoke r5.<java.util.Map$Entry: java.lang.Object getKey()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r17 = (org.apache.accumulo.core.data.Key) $r16
org.apache.accumulo.master.replication.WorkMaker.run : $r16 = interfaceinvoke r5.<java.util.Map$Entry: java.lang.Object getKey()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r17 = (org.apache.accumulo.core.data.Key) $r16 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : staticinvoke <org.apache.accumulo.core.replication.ReplicationSchema$StatusSection: void getFile(org.apache.accumulo.core.data.Key,org.apache.hadoop.io.Text)>($r17, $r14)
org.apache.accumulo.master.replication.WorkMaker.run : $r17 = (org.apache.accumulo.core.data.Key) $r16 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : staticinvoke <org.apache.accumulo.core.replication.ReplicationSchema$StatusSection: void getFile(org.apache.accumulo.core.data.Key,org.apache.hadoop.io.Text)>($r17, $r14) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r18 = interfaceinvoke r5.<java.util.Map$Entry: java.lang.Object getKey()>()
org.apache.accumulo.master.replication.WorkMaker.run : staticinvoke <org.apache.accumulo.core.replication.ReplicationSchema$StatusSection: void getFile(org.apache.accumulo.core.data.Key,org.apache.hadoop.io.Text)>($r17, $r14) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r18 = interfaceinvoke r5.<java.util.Map$Entry: java.lang.Object getKey()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r19 = (org.apache.accumulo.core.data.Key) $r18
org.apache.accumulo.master.replication.WorkMaker.run : $r18 = interfaceinvoke r5.<java.util.Map$Entry: java.lang.Object getKey()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r19 = (org.apache.accumulo.core.data.Key) $r18 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : r6 = staticinvoke <org.apache.accumulo.core.replication.ReplicationSchema$StatusSection: org.apache.accumulo.core.client.impl.Table$ID getTableId(org.apache.accumulo.core.data.Key)>($r19)
org.apache.accumulo.master.replication.WorkMaker.run : $r19 = (org.apache.accumulo.core.data.Key) $r18 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : r6 = staticinvoke <org.apache.accumulo.core.replication.ReplicationSchema$StatusSection: org.apache.accumulo.core.client.impl.Table$ID getTableId(org.apache.accumulo.core.data.Key)>($r19) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r20 = <org.apache.accumulo.master.replication.WorkMaker: org.slf4j.Logger log>
org.apache.accumulo.master.replication.WorkMaker.run : r6 = staticinvoke <org.apache.accumulo.core.replication.ReplicationSchema$StatusSection: org.apache.accumulo.core.client.impl.Table$ID getTableId(org.apache.accumulo.core.data.Key)>($r19) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r20 = <org.apache.accumulo.master.replication.WorkMaker: org.slf4j.Logger log> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r20.<org.slf4j.Logger: void debug(java.lang.String,java.lang.Object,java.lang.Object)>("Processing replication status record for {} on table {}", $r14, r6)
org.apache.accumulo.master.replication.WorkMaker.run : $r20 = <org.apache.accumulo.master.replication.WorkMaker: org.slf4j.Logger log> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r20.<org.slf4j.Logger: void debug(java.lang.String,java.lang.Object,java.lang.Object)>("Processing replication status record for {} on table {}", $r14, r6) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r34 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r20.<org.slf4j.Logger: void debug(java.lang.String,java.lang.Object,java.lang.Object)>("Processing replication status record for {} on table {}", $r14, r6) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r20.<org.slf4j.Logger: void debug(java.lang.String,java.lang.Object,java.lang.Object)>("Processing replication status record for {} on table {}", $r14, r6) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r21 = interfaceinvoke r5.<java.util.Map$Entry: java.lang.Object getValue()>()
org.apache.accumulo.master.replication.WorkMaker.run : $r21 = interfaceinvoke r5.<java.util.Map$Entry: java.lang.Object getValue()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r34 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r21 = interfaceinvoke r5.<java.util.Map$Entry: java.lang.Object getValue()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r21 = interfaceinvoke r5.<java.util.Map$Entry: java.lang.Object getValue()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r22 = (org.apache.accumulo.core.data.Value) $r21
org.apache.accumulo.master.replication.WorkMaker.run : $r22 = (org.apache.accumulo.core.data.Value) $r21 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r34 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r22 = (org.apache.accumulo.core.data.Value) $r21 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r22 = (org.apache.accumulo.core.data.Value) $r21 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r23 = virtualinvoke $r22.<org.apache.accumulo.core.data.Value: byte[] get()>()
org.apache.accumulo.master.replication.WorkMaker.run : $r23 = virtualinvoke $r22.<org.apache.accumulo.core.data.Value: byte[] get()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r34 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r23 = virtualinvoke $r22.<org.apache.accumulo.core.data.Value: byte[] get()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r23 = virtualinvoke $r22.<org.apache.accumulo.core.data.Value: byte[] get()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : r7 = staticinvoke <org.apache.accumulo.server.replication.proto.Replication$Status: org.apache.accumulo.server.replication.proto.Replication$Status parseFrom(byte[])>($r23)
org.apache.accumulo.master.replication.WorkMaker.run : r7 = staticinvoke <org.apache.accumulo.server.replication.proto.Replication$Status: org.apache.accumulo.server.replication.proto.Replication$Status parseFrom(byte[])>($r23) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r34 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : r7 = staticinvoke <org.apache.accumulo.server.replication.proto.Replication$Status: org.apache.accumulo.server.replication.proto.Replication$Status parseFrom(byte[])>($r23) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : r7 = staticinvoke <org.apache.accumulo.server.replication.proto.Replication$Status: org.apache.accumulo.server.replication.proto.Replication$Status parseFrom(byte[])>($r23) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : goto [?= $z1 = virtualinvoke r0.<org.apache.accumulo.master.replication.WorkMaker: boolean shouldCreateWork(org.apache.accumulo.server.replication.proto.Replication$Status)>(r7)]
org.apache.accumulo.master.replication.WorkMaker.run : goto [?= $z1 = virtualinvoke r0.<org.apache.accumulo.master.replication.WorkMaker: boolean shouldCreateWork(org.apache.accumulo.server.replication.proto.Replication$Status)>(r7)] >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $z1 = virtualinvoke r0.<org.apache.accumulo.master.replication.WorkMaker: boolean shouldCreateWork(org.apache.accumulo.server.replication.proto.Replication$Status)>(r7)
org.apache.accumulo.master.replication.WorkMaker.run : goto [?= $z1 = virtualinvoke r0.<org.apache.accumulo.master.replication.WorkMaker: boolean shouldCreateWork(org.apache.accumulo.server.replication.proto.Replication$Status)>(r7)] >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $z1 = virtualinvoke r0.<org.apache.accumulo.master.replication.WorkMaker: boolean shouldCreateWork(org.apache.accumulo.server.replication.proto.Replication$Status)>(r7) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : if $z1 != 0
org.apache.accumulo.master.replication.WorkMaker.run : $z1 = virtualinvoke r0.<org.apache.accumulo.master.replication.WorkMaker: boolean shouldCreateWork(org.apache.accumulo.server.replication.proto.Replication$Status)>(r7) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : if $z1 != 0 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r29 = <org.apache.accumulo.master.replication.WorkMaker: org.slf4j.Logger log>
org.apache.accumulo.master.replication.WorkMaker.run : if $z1 != 0 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r24 = r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.server.AccumuloServerContext context>
org.apache.accumulo.master.replication.WorkMaker.run : if $z1 != 0 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r24 = r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.server.AccumuloServerContext context> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r25 = virtualinvoke $r24.<org.apache.accumulo.server.AccumuloServerContext: org.apache.accumulo.server.conf.ServerConfigurationFactory getServerConfigurationFactory()>()
org.apache.accumulo.master.replication.WorkMaker.run : $r24 = r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.server.AccumuloServerContext context> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r25 = virtualinvoke $r24.<org.apache.accumulo.server.AccumuloServerContext: org.apache.accumulo.server.conf.ServerConfigurationFactory getServerConfigurationFactory()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : r42 = virtualinvoke $r25.<org.apache.accumulo.server.conf.ServerConfigurationFactory: org.apache.accumulo.server.conf.TableConfiguration getTableConfiguration(org.apache.accumulo.core.client.impl.Table$ID)>(r6)
org.apache.accumulo.master.replication.WorkMaker.run : $r25 = virtualinvoke $r24.<org.apache.accumulo.server.AccumuloServerContext: org.apache.accumulo.server.conf.ServerConfigurationFactory getServerConfigurationFactory()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : r42 = virtualinvoke $r25.<org.apache.accumulo.server.conf.ServerConfigurationFactory: org.apache.accumulo.server.conf.TableConfiguration getTableConfiguration(org.apache.accumulo.core.client.impl.Table$ID)>(r6) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : if null != r42
org.apache.accumulo.master.replication.WorkMaker.run : r42 = virtualinvoke $r25.<org.apache.accumulo.server.conf.ServerConfigurationFactory: org.apache.accumulo.server.conf.TableConfiguration getTableConfiguration(org.apache.accumulo.core.client.impl.Table$ID)>(r6) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : if null != r42 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : goto [?= $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()]
org.apache.accumulo.master.replication.WorkMaker.run : if null != r42 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : r43 = virtualinvoke r0.<org.apache.accumulo.master.replication.WorkMaker: java.util.Map getReplicationTargets(org.apache.accumulo.server.conf.TableConfiguration)>(r42)
org.apache.accumulo.master.replication.WorkMaker.run : if null != r42 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : r43 = virtualinvoke r0.<org.apache.accumulo.master.replication.WorkMaker: java.util.Map getReplicationTargets(org.apache.accumulo.server.conf.TableConfiguration)>(r42) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $z2 = interfaceinvoke r43.<java.util.Map: boolean isEmpty()>()
org.apache.accumulo.master.replication.WorkMaker.run : r43 = virtualinvoke r0.<org.apache.accumulo.master.replication.WorkMaker: java.util.Map getReplicationTargets(org.apache.accumulo.server.conf.TableConfiguration)>(r42) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $z2 = interfaceinvoke r43.<java.util.Map: boolean isEmpty()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : if $z2 != 0
org.apache.accumulo.master.replication.WorkMaker.run : $z2 = interfaceinvoke r43.<java.util.Map: boolean isEmpty()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : if $z2 != 0 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : r8 = staticinvoke <org.apache.accumulo.core.trace.Trace: org.apache.accumulo.core.trace.Span start(java.lang.String)>("createWorkMutations")
org.apache.accumulo.master.replication.WorkMaker.run : if $z2 != 0 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r26 = <org.apache.accumulo.master.replication.WorkMaker: org.slf4j.Logger log>
org.apache.accumulo.master.replication.WorkMaker.run : if $z2 != 0 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r26 = <org.apache.accumulo.master.replication.WorkMaker: org.slf4j.Logger log> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r26.<org.slf4j.Logger: void warn(java.lang.String,java.lang.Object)>("No configured targets for table with ID {}", r6)
org.apache.accumulo.master.replication.WorkMaker.run : $r26 = <org.apache.accumulo.master.replication.WorkMaker: org.slf4j.Logger log> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r26.<org.slf4j.Logger: void warn(java.lang.String,java.lang.Object)>("No configured targets for table with ID {}", r6) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : goto [?= $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()]
org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r26.<org.slf4j.Logger: void warn(java.lang.String,java.lang.Object)>("No configured targets for table with ID {}", r6) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : goto [?= $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()] >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()
org.apache.accumulo.master.replication.WorkMaker.run : goto [?= $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()] >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : r8 = staticinvoke <org.apache.accumulo.core.trace.Trace: org.apache.accumulo.core.trace.Span start(java.lang.String)>("createWorkMutations") >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r38 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : r8 = staticinvoke <org.apache.accumulo.core.trace.Trace: org.apache.accumulo.core.trace.Span start(java.lang.String)>("createWorkMutations") >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : r8 = staticinvoke <org.apache.accumulo.core.trace.Trace: org.apache.accumulo.core.trace.Span start(java.lang.String)>("createWorkMutations") >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r27 = interfaceinvoke r5.<java.util.Map$Entry: java.lang.Object getValue()>()
org.apache.accumulo.master.replication.WorkMaker.run : $r27 = interfaceinvoke r5.<java.util.Map$Entry: java.lang.Object getValue()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r28 = (org.apache.accumulo.core.data.Value) $r27
org.apache.accumulo.master.replication.WorkMaker.run : $r27 = interfaceinvoke r5.<java.util.Map$Entry: java.lang.Object getValue()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r38 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r8.<org.apache.accumulo.core.trace.Span: void stop()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : throw $r38
org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r8.<org.apache.accumulo.core.trace.Span: void stop()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : throw $r38 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r28 = (org.apache.accumulo.core.data.Value) $r27 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r0.<org.apache.accumulo.master.replication.WorkMaker: void addWorkRecord(org.apache.hadoop.io.Text,org.apache.accumulo.core.data.Value,java.util.Map,org.apache.accumulo.core.client.impl.Table$ID)>($r14, $r28, r43, r6)
org.apache.accumulo.master.replication.WorkMaker.run : $r28 = (org.apache.accumulo.core.data.Value) $r27 >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r38 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r0.<org.apache.accumulo.master.replication.WorkMaker: void addWorkRecord(org.apache.hadoop.io.Text,org.apache.accumulo.core.data.Value,java.util.Map,org.apache.accumulo.core.client.impl.Table$ID)>($r14, $r28, r43, r6) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r38 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r0.<org.apache.accumulo.master.replication.WorkMaker: void addWorkRecord(org.apache.hadoop.io.Text,org.apache.accumulo.core.data.Value,java.util.Map,org.apache.accumulo.core.client.impl.Table$ID)>($r14, $r28, r43, r6) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r0.<org.apache.accumulo.master.replication.WorkMaker: void addWorkRecord(org.apache.hadoop.io.Text,org.apache.accumulo.core.data.Value,java.util.Map,org.apache.accumulo.core.client.impl.Table$ID)>($r14, $r28, r43, r6) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r8.<org.apache.accumulo.core.trace.Span: void stop()>()
org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r8.<org.apache.accumulo.core.trace.Span: void stop()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : goto [?= (branch)]
org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r8.<org.apache.accumulo.core.trace.Span: void stop()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : goto [?= (branch)] >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : goto [?= (branch)]
org.apache.accumulo.master.replication.WorkMaker.run : goto [?= (branch)] >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : goto [?= (branch)] >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : goto [?= $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()]
org.apache.accumulo.master.replication.WorkMaker.run : goto [?= (branch)] >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : goto [?= $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()] >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()
org.apache.accumulo.master.replication.WorkMaker.run : goto [?= $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()] >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r29 = <org.apache.accumulo.master.replication.WorkMaker: org.slf4j.Logger log> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r30 = virtualinvoke r7.<org.apache.accumulo.server.replication.proto.Replication$Status: java.lang.String toString()>()
org.apache.accumulo.master.replication.WorkMaker.run : $r29 = <org.apache.accumulo.master.replication.WorkMaker: org.slf4j.Logger log> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r30 = virtualinvoke r7.<org.apache.accumulo.server.replication.proto.Replication$Status: java.lang.String toString()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r29.<org.slf4j.Logger: void debug(java.lang.String,java.lang.Object)>("Not creating work: {}", $r30)
org.apache.accumulo.master.replication.WorkMaker.run : $r30 = virtualinvoke r7.<org.apache.accumulo.server.replication.proto.Replication$Status: java.lang.String toString()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r29.<org.slf4j.Logger: void debug(java.lang.String,java.lang.Object)>("Not creating work: {}", $r30) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : goto [?= $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()]
org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r29.<org.slf4j.Logger: void debug(java.lang.String,java.lang.Object)>("Not creating work: {}", $r30) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : goto [?= $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()] >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()
org.apache.accumulo.master.replication.WorkMaker.run : goto [?= $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()] >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r35 = <org.apache.accumulo.master.replication.WorkMaker: org.slf4j.Logger log> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r35.<org.slf4j.Logger: void error(java.lang.String,java.lang.Object,java.lang.Object)>("Could not parse protobuf for {} from table {}", $r14, r6)
org.apache.accumulo.master.replication.WorkMaker.run : $r35 = <org.apache.accumulo.master.replication.WorkMaker: org.slf4j.Logger log> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r35.<org.slf4j.Logger: void error(java.lang.String,java.lang.Object,java.lang.Object)>("Could not parse protobuf for {} from table {}", $r14, r6) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : goto [?= $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()]
org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r35.<org.slf4j.Logger: void error(java.lang.String,java.lang.Object,java.lang.Object)>("Could not parse protobuf for {} from table {}", $r14, r6) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : goto [?= $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()] >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()
org.apache.accumulo.master.replication.WorkMaker.run : goto [?= $z3 = interfaceinvoke r4.<java.util.Iterator: boolean hasNext()>()] >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r31 = r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.core.client.Connector conn> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r36 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r31 = r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.core.client.Connector conn> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r31 = r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.core.client.Connector conn> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r32 = staticinvoke <org.apache.accumulo.core.replication.ReplicationTable: org.apache.accumulo.core.client.BatchWriter getBatchWriter(org.apache.accumulo.core.client.Connector)>($r31)
org.apache.accumulo.master.replication.WorkMaker.run : $r32 = staticinvoke <org.apache.accumulo.core.replication.ReplicationTable: org.apache.accumulo.core.client.BatchWriter getBatchWriter(org.apache.accumulo.core.client.Connector)>($r31) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r36 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r32 = staticinvoke <org.apache.accumulo.core.replication.ReplicationTable: org.apache.accumulo.core.client.BatchWriter getBatchWriter(org.apache.accumulo.core.client.Connector)>($r31) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : $r32 = staticinvoke <org.apache.accumulo.core.replication.ReplicationTable: org.apache.accumulo.core.client.BatchWriter getBatchWriter(org.apache.accumulo.core.client.Connector)>($r31) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r0.<org.apache.accumulo.master.replication.WorkMaker: void setBatchWriter(org.apache.accumulo.core.client.BatchWriter)>($r32)
org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r0.<org.apache.accumulo.master.replication.WorkMaker: void setBatchWriter(org.apache.accumulo.core.client.BatchWriter)>($r32) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r36 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r0.<org.apache.accumulo.master.replication.WorkMaker: void setBatchWriter(org.apache.accumulo.core.client.BatchWriter)>($r32) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r0.<org.apache.accumulo.master.replication.WorkMaker: void setBatchWriter(org.apache.accumulo.core.client.BatchWriter)>($r32) >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : goto [?= staticinvoke <org.apache.accumulo.core.replication.ReplicationSchema$StatusSection: void limit(org.apache.accumulo.core.client.ScannerBase)>(r2)]
org.apache.accumulo.master.replication.WorkMaker.run : $r37 = <org.apache.accumulo.master.replication.WorkMaker: org.slf4j.Logger log> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r37.<org.slf4j.Logger: void warn(java.lang.String)>("Replication table was online, but not anymore")
org.apache.accumulo.master.replication.WorkMaker.run : $r37 = <org.apache.accumulo.master.replication.WorkMaker: org.slf4j.Logger log> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r37.<org.slf4j.Logger: void warn(java.lang.String)>("Replication table was online, but not anymore") >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.core.client.BatchWriter writer> = null
org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r37.<org.slf4j.Logger: void warn(java.lang.String)>("Replication table was online, but not anymore") >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.core.client.BatchWriter writer> = null >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r1.<org.apache.accumulo.core.trace.Span: void stop()>()
org.apache.accumulo.master.replication.WorkMaker.run : r0.<org.apache.accumulo.master.replication.WorkMaker: org.apache.accumulo.core.client.BatchWriter writer> = null >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : $r39 := @caughtexception
org.apache.accumulo.master.replication.WorkMaker.run : virtualinvoke r1.<org.apache.accumulo.core.trace.Span: void stop()>() >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : return
org.apache.accumulo.master.replication.WorkMaker.run : $r33 = <org.apache.accumulo.master.replication.WorkMaker: org.slf4j.Logger log> >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r33.<org.slf4j.Logger: void debug(java.lang.String)>("Replication table is not yet online")
org.apache.accumulo.master.replication.WorkMaker.run : interfaceinvoke $r33.<org.slf4j.Logger: void debug(java.lang.String)>("Replication table is not yet online") >>>>>>>> org.apache.accumulo.master.replication.WorkMaker.run : return
